packages/web3-eth-ens/test/fixtures/ens/bytecode/DummyNameWrapperBytecode.ts

```diff
@@ -0,0 +1,18 @@
+/*
+This file is part of web3.js.
+
+web3.js is free software: you can redistribute it and/or modify
+it under the terms of the GNU Lesser General Public License as published by
+the Free Software Foundation, either version 3 of the License, or
+(at your option) any later version.
+
+web3.js is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU Lesser General Public License for more details.
+
+You should have received a copy of the GNU Lesser General Public License
+along with web3.js.  If not, see <http://www.gnu.org/licenses/>.
+*/
+export const DummyNameWrapperBytecode =
```

Comment:
Also could you add solidity source link for this in comments, 
will be easier to debug in future if required.

Reply:
And is this only used for testing?
If yes, what about moving it to test/fixtures?
If not, what about renaming it to something like BasicNameWrapperApi or simply NameWrapperApi?

Reply:
+1